novendor.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -eu
```

Comment:
```suggestion
set -euo pipefail
```

To trap failures inside pipes.

Reply:
:+1: